//2. Handle the Arithmetic exception using try-catch block


public class A2_HandleArithmetic {
    public static void main(String[] args) {
        try {
            int result = 10 / 0;
        } catch (ArithmeticException e) {
            System.out.println("Caught ArithmeticException: " + e.getMessage());
        }
    }
}
